packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -32,22 +32,24 @@
 .ds-c-usa-banner__header {
   // Applying display:grid to account for flag icon alignment with text elements in mobile/table viewports
   display: grid;
-  gap: 0.5ch 1ch;
+  gap: 2px 1ch;
   grid-template-areas:
     'icon text'
-    'icon action';
+    'spacer action';
   grid-template-columns: 1.5em auto;
+  grid-template-rows: min-content min-content;
   line-height: var(--heading__line-height);
   min-height: 3rem;
-  padding-block: 0.25rem;
+  padding-block: 0.5rem;
   position: relative;
 
   // Need to target viewport 1px larger than $media-width-md to ensure the layout is correct on the breakpoint
-  @media (max-width: ($media-width-sm + 1)) {
+  @media (max-width: ($media-width-md + 1)) {
     &.ds-c-usa-banner__header--expanded {
       // Additional padding needed to accommodate close button in expanded state
+      // Padding is .ds-c-usa-banner__button-icon-container's width + .25rem for "gutter" between content and button
       // Button is only visible in mobile/tablet viewports
-      padding-inline-end: 3.5rem;
+      padding-inline-end: 3.25rem;
```

Comment:
Is this `0.25rem` for the `2px` defined above? If that's the case, should we actually change `2px` to `0.25rem` to keep the gap relative to the text size?

Reply:
no, i made this adjustment because the gap between the close button and text looked too wide at certain breakpoints. the button's 3rem's wide, so this essentially created a .25rem (4px) gap between text and button. i can certainly change the other 2px gap to 0.125rem.

Reply:
gonna hold off on making this change - design changed and is currently unapproved